pkgs/top-level/all-packages.nix

```
@@ -13448,23 +13446,23 @@ with pkgs;
   };
 
   firefox-beta-bin-unwrapped = firefox-bin-unwrapped.override {
+    applicationName = firefox-beta-unwrapped.passthru.applicationName;
     channel = "beta";
     generated = import ../applications/networking/browsers/firefox-bin/beta_sources.nix;
   };
 
   firefox-beta-bin = res.wrapFirefox firefox-beta-bin-unwrapped {
     pname = "firefox-beta-bin";
-    desktopName = "Firefox Beta";
   };
 
   firefox-devedition-bin-unwrapped = callPackage ../applications/networking/browsers/firefox-bin {
+    applicationName = firefox-devedition-unwrapped.passthru.applicationName;
```

```suggestion
    inherit (firefox-devedition-unwrapped.passthru) applicationName;
```
same can be done to multiple ocures above